specification/schema/featureTable.schema.json

```diff
@@ -67,25 +67,13 @@
                 "byteOffset"
             ]
         },
-        "numericArray": {
-            "type": "array",
-            "items": {
-                "type": "number"
-            }
-        },
         "property": {
             "title": "Property",
-            "description": "A user-defined property which specifies per-feature application-specific metadata in a tile. Values either can be defined directly in the JSON as an array, or can refer to sections in the binary body with a `BinaryBodyReference` object.",
+            "description": "A user-defined property which specifies application-specific metadata in a tile. Values can refer to sections in the binary body with a `BinaryBodyReference` object. Global values can be also be defined directly in the JSON.",
             "oneOf": [
                 {
                     "$ref": "#/definitions/binaryBodyReference"
                 },
-                {
-                    "type": "array",
-                    "items": {
-                        "type": "number"
-                    }
-                },
                 {
                     "type": "number"
                 }
```

Comment:
> The schema supported storing arrays in the header. Only the *text* said that these arrays were not allowed to be **per-feature** values. But these arrays may have served the purpose of having a **global** "vector" or "matrix" (represented as an array). Now, the only way to store a global array-typed value is with a `binaryBodyReference`...

To avoid this problem should a `property` be one of the existing types?

```json
"oneOf": [
  {
    "$ref": "#/definitions/binaryBodyReference"
  },
  {
    "$ref": "#/definitions/globalPropertyBoolean"
  },
  {
    "$ref": "#/definitions/globalPropertyInteger"
  },
  {
    "$ref": "#/definitions/globalPropertyNumber"
  },
  {
    "$ref": "#/definitions/globalPropertyCartesian3"
  },
  {
    "$ref": "#/definitions/globalPropertyCartesian4"
  }
]
```

Reply:
There's no `globalPropertyMatrix4` but that could be ok.

Reply:
I have to admit that I may not have thoroughly understood some of the design decisions, that led to the current structure, and am now wondering about things like

- why is the `globalPorpertyBoolean` not allowed to be defined with a `byteOffset` (like the `...Integer` and `...Number` counterparts)? (Is this related to the data type size?)
- Why do `globalPropertyCartesian3`/`4` extend `"$ref": "tilesetProperty.schema.json"`? (Is this because they are "structured" (and not _primitive_) types?)

but that may not be utterly important right now.

The last suggestion (of listing the `binaryBodyReference` and the `global...` ones explicitly) could be one way of solving this. Assuming that this does not have undesired side-effects that I'm currently overlooking, I'll update this PR accordingly (early tomorrow) and mark it as ready.

Reply:
> - why is the `globalPropertyBoolean` not allowed to be defined with a `byteOffset` (like the `...Integer` and `...Number` counterparts)? (Is this related to the data type size?)

Most likely because we didn't specify a data type for booleans.

- Why do `globalPropertyCartesian3`/`4` extend `"$ref": "tilesetProperty.schema.json"`? (Is this because they are "structured" (and not _primitive_) types?)

Yes, every object type inherits `tilesetProperty.schema.json` so it can have extras/extensions.

Reply:
The changes here are good for the known feature table implementations - b3dm, i3dm, and pnts - and with glTF around the corner I don't think we have to focus too much on the smaller details.